Interfaces/API/Dirac.py

```diff
@@ -2029,7 +2029,7 @@ def selectJobs( self, status = None, minorStatus = None, applicationStatus = Non
     jobIDs = result['Value']
     self.log.verbose( '%s job(s) selected' % ( len( jobIDs ) ) )
     if not jobIDs:
-      return S_ERROR( 'No jobs selected for conditions: %s' % conditions )
+      return S_ERROR( "No jobs selected with date '%s' for conditions: %s" % (str(date), conditions) )
```

Comment:
I would rather do the following:
```
self.log.error( "No jobs selected", "with date '%s' for conditions: %s" % (str(date), conditions))
return S_ERROR( "No jobs selected" )
```

Because this would keep the return message cleaner.